packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java

```diff
@@ -182,9 +190,6 @@ public void onMethodCall(MethodCall call, final Result result) {
             camera.close();
           }
           camera = new Camera(cameraName, resolutionPreset, result);
-          this.activity
```

Comment:
The reason we `registerActivityLifecycleCallbacks` here is because we unregister them in the `dispose` method call on L214. I would recommend keeping this and just calling `orientationEventListener.enable();` here.